docs2/site/docs/migrations/migration7.md

```diff
@@ -290,6 +290,53 @@ Please note that the `SchemaExporter` class will export type defintions as type
 
 Please see the `SchemaExtensions.PrintAsync` source code implementation specifics.
 
+### 14. Infer field nullability from NRT annotations
+
+Since 7.9 you can enable the inference of the field nullability from the Null Reference Types (NRT)
+annotations when defining the field with expression. To enable the feature set the  
+`GlobalSwitches.InferFieldNullabilityFromNRTAnnotations` to `true`.  
+
+For example, given the following code
+
+```c#
+public class Person
+{
+    public string FullName { get; set; }
+    public string? SpouseName { get; set; }
+    public IList<string>? Children { get; set; }
+}
+
+public class PersonGraphType : ObjectGraphType<Person>
+{
+    public PersonGraphType()
+    {
+        Field(p => p.FullName);
+        Field(p => p.SpouseName);
+        Field(p => p.Children);
+    }
+}
+```
+
+When `InferFieldNullabilityFromNRTAnnotations` is `false` (default), the result is:
+
+```graphql
+type Person {
+  fullName: String!
+  spouseName: String!
+  children: [String]!
+}
+```
+
+When `InferFieldNullabilityFromNRTAnnotations` is `true`:
+
+```graphql
+type Person {
+  fullName: String!
+  spouseName: String
+  children: [String!]
+}
+```
+
```

Comment:
@Shane32 please review migration notes update

Reply:
Migration notes look perfect! I do suggest we move/copy these to the 8.0 migration notes in a separate PR targeting develop, along with changing the default value and appropriate changes to the notes to reflect this. 

This really is more of a new feature for 8.0, backported to 7.9, and I’d like to emphasis this feature in the new features section of 8.0. Decent chance are that both 7.9 and 8.0 will be released together. Or that any user upgrading to 7.9 is probably just doing so because of dependabot and not reading any migration notes since they are already on 7.x. 

Sometimes I do add minor version new features to the migration notes but usually only when it’s early in the major version’s lifecycle.